api/application/client.go

```diff
@@ -319,33 +319,44 @@ func (c *Client) DestroyUnitsDeprecated(unitNames ...string) error {
 	return c.facade.FacadeCall("DestroyUnits", params, nil)
 }
 
+// DestroyUnitsParams contains parameters for the DestroyUnits API method.
+type DestroyUnitsParams struct {
```

Comment:
This doesn't seem related to the PR description. Maybe better to keep the PR more focused?